### DISCUSSION TITLE
Workflows Queued, not running

### DISCUSSION THREAD

**Discussion by danielgary** [community/community#39257] [Actions] [answered]
### Select Topic Area

Bug

### Body

In my public repo, https://github.com/danielgary/nextjs-app/actions
my actions have been queued and are not running.  I checked github status and it shows all systems operational.  Just wondering if there was something wrong?

**Comment by danielgary** [comment, marked as answer]
Never mind I guess.  After ten minutes being queued they finally ran.

**Reply by i-rember** [reply]
Wish that happened to me. They've been queued for HOURS now.
